<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 197, 255);</string>
  </property>
  <widget class="QPushButton" name="B_Conf_Conta">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>510</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(113, 229, 231);</string>
   </property>
   <property name="text">
    <string>Configurações de conta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_NovoObjetivo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Meus Objetivos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(198, 255, 253);</string>
   </property>
   <property name="text">
    <string>Novo Objetivo</string>
   </property>
  </widget>
  <widget class="QLabel" name="Img_User">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/USUARIO.png);
background-color: rgb(113, 229, 231);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Back_Lista">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(113, 229, 231);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="B_Sair">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(113, 229, 231);</string>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_MeusObjetivos">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Meus Objetivos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(198, 255, 253);</string>
   </property>
   <property name="text">
    <string>Meus Objetivos</string>
   </property>
  </widget>
  <widget class="QLabel" name="NomeUsuario">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(113, 229, 231);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_CAD_OBJ">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; font-style:italic;&quot;&gt;Editando Objetivos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_back_meio">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>451</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(113, 229, 231);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="B_cadastrar">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>530</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(113, 229, 231);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>EDITAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_materia">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(113, 229, 231);</string>
   </property>
   <property name="text">
    <string>Matéria:</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_assunto">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(113, 229, 231);</string>
   </property>
   <property name="text">
    <string>Assunto:</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_objetivo1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(113, 229, 231);</string>
   </property>
   <property name="text">
    <string>Objetivo:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="C_materia">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>79</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="C_assunto">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="C_Objetivo1">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_RC_A1">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>210</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(113, 229, 231);</string>
   </property>
   <property name="text">
    <string>Resultado-Chave a</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="C_RC_A1">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <zorder>Back_Lista</zorder>
  <zorder>B_Conf_Conta</zorder>
  <zorder>B_NovoObjetivo</zorder>
  <zorder>Img_User</zorder>
  <zorder>B_Sair</zorder>
  <zorder>B_MeusObjetivos</zorder>
  <zorder>NomeUsuario</zorder>
  <zorder>L_CAD_OBJ</zorder>
  <zorder>L_back_meio</zorder>
  <zorder>B_cadastrar</zorder>
  <zorder>L_materia</zorder>
  <zorder>L_assunto</zorder>
  <zorder>L_objetivo1</zorder>
  <zorder>C_materia</zorder>
  <zorder>C_assunto</zorder>
  <zorder>C_Objetivo1</zorder>
  <zorder>L_RC_A1</zorder>
  <zorder>C_RC_A1</zorder>
 </widget>
 <resources>
  <include location="sourcee.qrc"/>
 </resources>
 <connections/>
</ui>
